charts/certmanager/templates/cert-manager-installation.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
After working with the VP framework more and more. For the case of simplicity, what about delegating the actual installation of the operator similar to what is being accomplished for [RHBK](https://github.com/validatedpatterns/layered-zero-trust/blob/main/values-hub.yaml#L11-L16).

Reply:
Completed the updates, the changes are incorporated to delegate the installation to VP framework.